centinel/primitives/http.py

```diff
@@ -49,10 +52,64 @@ def get_request(host, path="/", headers=None, ssl=False,
     }
     # the external result is used when threading to store
     # the results in the list container provided.
-    if external is not None and type(external) is dict:
-        external[url] = result
+    # if external is not None and type(external) is dict:
```

Comment:
It doesn't seem like external is being used here. We should either add the lines back in or delete the lines and the function argument to avoid confusion.